sample/ssl_client1.c

```diff
@@ -0,0 +1,315 @@
+/*
```

Comment:
so this is mbedtls example, then let's reflect this in filename?

Reply:
I have rename it to `ssl-client-mbedtls`